### PR TITLE
Add sign message/data

### PR DESCRIPTION
**Pull request by hewigovens**

### CODE REVIEW

File: Example/TrustSDK/ViewController.swift [hunk 1]
```diff
@@ -48,16 +49,34 @@ class ViewController: UIViewController {
             }
         }
     }
-    
+
+    @IBAction func signEthereumMessage(_ sender: UIButton) {
+        let data = Data("Some message".utf8)
+        let message = Data("\u{19}Ethereum Signed Message:\n\(data.count)".utf8)
```
Comment (vikmeup) [inline Example/TrustSDK/ViewController.swift:55]: can we handle this on app side to avoid adding `CryptoSwift` lib?

Comment (vikmeup) [reply Example/TrustSDK/ViewController.swift:55]: does iOS crypto supports sha3?

Comment (hewigovens) [reply Example/TrustSDK/ViewController.swift:55]: This is the test **Example** app, we don't need it in Trust

Comment (hewigovens) [reply Example/TrustSDK/ViewController.swift:55]: no sha3 in CryptoKit :(

**Pull request merged**